include/odp/api/spec/traffic_mngr.h

```diff
@@ -794,20 +794,32 @@ typedef enum {
  */
 typedef struct {
 	/** The committed information rate for this shaper profile.  The units
-	 * for this integer are always in bits per second. */
-	uint64_t commit_bps;
+	 * for this integer is in bits per second when packet_mode is
+	 * not TRUE while packets per second when packet mode is TRUE.
+	 */
+	union {
+		uint64_t commit_bps; /**< Commit information rate in bps */
```

Comment:
This one and also the peak_bps field should by marked deprecated. Or if backward compatibility for this is not needed, then the old field could just be dropped.

Reply:
I have already marked it as deprecated in last patch. I can remove it in last patch if you think it is not necessary to be backward compatible. Since it is being used in validation tests, I had to do it after changing the validation tests.

Reply:
Sorry, I missed the deprecation in the last patch.